app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/MainViewModel.kt

```diff
@@ -14,6 +14,14 @@ class MainViewModel : ViewModel() {
     private var _notification = Notification()
     private var _notifications : MutableLiveData<ArrayList<Notification>> = MutableLiveData<ArrayList<Notification>>()
 
+    internal var notification: Notification
+        get() {return _notification}
+        set(value) {_notification = value}
+
+    internal var notifications: MutableLiveData<ArrayList<Notification>>
+        get() {return _notifications}
+        set(value) {_notifications = value}
+
```

Comment:
I simply moved these variable declaration statements to the top of your class, per Kotlin conventions.